config.cmake

```diff
@@ -302,12 +302,3 @@ hunter_config(Lua VERSION 5.1.5-p3)
 hunter_config(SDL2 VERSION 2.0.4-urho-p4)
 
 # }
-
-# FIXME, waiting for release:
-# * https://github.com/jarro2783/cxxopts/issues/110#issuecomment-394909564
```

Comment:
Moved to Hunter:
* https://github.com/ruslo/hunter/blob/50330c772ad57eeb6495b2d61985e57eeaa450e4/cmake/projects/cxxopts/hunter.cmake#L111-L122